src/components/App.js

```diff
@@ -18,19 +19,31 @@ class App extends Component {
       searchInput,
       allCategories,
       selectedCategories,
+      chosenCategory,
+      events,
       setSearchKeyword,
+      getCategories,
+      getEvents,
     } = this.props;
 
     return (
       <div className="App">
         <SearchInput {...this.props} label="Search Events"/>
-          <Button
-            className='hello'
-            aria-label='hello'
-            content='hello'
-            handleClick={ () => setSearchKeyword(searchInput) }
-          />
-          <CategoriesList allCategories={allCategories} selectedCategories={selectedCategories} />
+        <Button
+          className='hello'
+          aria-label='hello'
+          content='hello'
+          handleClick={ () => setSearchKeyword(searchInput) }
+        />
+        <CategoriesList allCategories={allCategories} selectedCategories={selectedCategories} />
```

Comment:
the merged category routing PR will have conflicts here (this should be {childrenWithProps} which will automatically plug in CategoriesList or CategoryPage based on page route url)